### DISCUSSION TITLE
How to disable auto-close the Neo-tree window after moving window from git_status

### DISCUSSION THREAD

**Discussion by kay-adamof** [nvim-neo-tree/neo-tree.nvim#579] [Q&A] [answered]
## My issue

- Open up "git_status" tab in Neo-tree
- Select a file and hit enter to open it in nvim window
- Neotree is `auto-closed` (But I don't wanna it ! )

When the cursor is in `filesystem` or `buffers` in neotree, auto-closing neotree is not run. However, when only in `git_status`, pressing a file name will open up that file but neotree will be closed automatically.

Is there some settings to work around this issue?

**Comment by cseickel** [comment]
It's not supposed to do that. Can you share your neo-tree config?

**Comment by kay-adamof** [comment, marked as answer]
@cseickel 
You're right, I made a mistake in my config.

After your reply, I checked my config and found where the issue is occurred.

```
  git_status = {
    window = {
      position = "float", <-- THIS !! 
    },
  },
```

Thank you for pointing out and your effort.
